app/controllers/admin/registrations_controller.rb

```diff
@@ -6,12 +6,17 @@ class RegistrationsController < Admin::BaseController
 
     def index
       authorize! :show, Registration.new(conference_id: @conference.id)
-      @pdf_filename = "#{@conference.title}.pdf"
-      @registrations = @conference.registrations.includes(:user).order('registrations.created_at ASC')
-      @attended = @conference.registrations.where('attended = ?', true).count
+      @registrations = @conference.registrations.eager_load(
+        :qanswers, user: :roles
+      ).order('registrations.created_at ASC').to_a
+      @attended = @registrations.count(&:attended)
+      @questions = @conference.questions.to_a
```

Comment:
why is this variable needed? :thinking:

Reply:
Readability.

Reply:
When we iterate over the questions, it caused ActiveRecord's enumberable overrides to perform additional (unnecessary) queries.